test/indexSpec.js

```diff
@@ -24,6 +24,10 @@ describe('constructor', () => {
     assert.isObject(new PluginVirtual(options))
   })
 
+  it('should export the Token methods from the module', () => {
```

Comment:
The token functions are already tested in the other specs, and because of the fixtures like prefixes and public keys, the tests should break if the Token behavior changes.